src/components/DocumentManager/hooks/DocumentManagerState.js

```diff
@@ -19,7 +19,7 @@ function useDocumentManagerState(collectionId) {
         setSelectedDocument(documentWithFile);
       }
     })();
-  }, []);
+  }, [collectionId]);
```

Comment:
It's important to list all dependencies. The new version of react-scripts (that I upgraded to) has linting rules built in to catch this. If you remove this change on this branch, you'll see the linting error it shows.

Reply:
Completed